proxy/http/HttpTransact.h

```diff
@@ -154,6 +154,7 @@ enum ViaString_t {
   VIA_ERROR_CACHE_READ        = 'R',
   VIA_ERROR_MOVED_TEMPORARILY = 'M',
   VIA_ERROR_LOOP_DETECTED     = 'L',
+  VIA_ERROR_UNKNOWN           = ' ',
```

Comment:
The blank space value for this come from traffic_via:
https://github.com/apache/trafficserver/blob/4623209d348e1ca93c668684bb9ba3acfbbf60d5/src/traffic_via/traffic_via.cc#L176